changelog/b6227.dd

```diff
@@ -1,24 +1,21 @@
-Comparison of values belonging to different enums is deprecated.
+Comparison that takes into account the enumerated type improves code correctness by
+disallowing comparison of unrelated enumerated types. 
```

Comment:
Trailing whitespace. Also can changelog titles be several lines? README says no @wilzbach